pkgs/top-level/python-packages.nix

```
@@ -1904,6 +1904,8 @@ in {
 
   cvxopt = callPackage ../development/python-modules/cvxopt { };
 
+  cvxpy = callPackage ../development/python-modules/cvxpy { scs = self.scs; };
```

uneeded, default behavior of callPackage
```suggestion
  cvxpy = callPackage ../development/python-modules/cvxpy { };
```

also causing error:

```
error: while querying the derivation named 'python3.7-cvxpy-1.0.25':
while evaluating the attribute 'out.outPath' at /home/jon/.cache/nixpkgs-review/pr-78898/nixpkgs/lib/customisation.nix:151:13:
while evaluating the attribute 'propagatedBuildInputs' of the derivation 'python3.7-cvxpy-1.0.25' at /home/jon/.cache/nixpkgs-review/pr-78898/nixpkgs/pkgs/development/interpreters/python/mk-python-derivation.nix:105:3:
while evaluating 'getOutput' at /home/jon/.cache/nixpkgs-review/pr-78898/nixpkgs/lib/attrsets.nix:464:23, called from undefined position:
while evaluating anonymous function at /home/jon/.cache/nixpkgs-review/pr-78898/nixpkgs/pkgs/stdenv/generic/make-derivation.nix:156:17, called from undefined position:
attribute 'scs' missing, at /home/jon/.cache/nixpkgs-review/pr-78898/nixpkgs/pkgs/top-level/python-packages.nix:1907:67
```